### PR TITLE
Resolve flake8 import related violations

### PR DESCRIPTION
**Pull request by cjmayo**
A mix of unused or out of date code and some simple line moves to resolve E402 and F401 violations.

### CODE REVIEW

File: linkcheck/colorama.py [hunk 1]
```diff
@@ -26,14 +26,13 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from ctypes import (windll, byref, Structure, c_char, c_short, c_uint32,
+                    c_ushort, ArgumentError, WinError)
```
Comment (mgedmin) [inline linkcheck/colorama.py:30]: Hey hey hey -- this is _not_ related to shit PR, but! -- hey! HEY!

I recently (re)learned about the existence of https://pypi.org/project/colorama/, and this looks like a vendored copy?

Can we pull it in via a dependency instead?  (Out of scope for this PR, obviously).

File: linkcheck/configuration/__init__.py [hunk 2]
```diff
@@ -552,7 +552,7 @@ def get_kde_home_dir():
             return
         kde3_home = os.path.join(home, ".kde")
         kde4_home = os.path.join(home, ".kde4")
-        if fileutil.find_executable("kde4-config"):
+        if shutil.which("kde4-config"):
```
Comment (mgedmin) [inline linkcheck/configuration/__init__.py:555]: Go Python 3!

File: linkcheck/colorama.py (hunk 1 above)
Comment (mgedmin) [reply linkcheck/colorama.py:30]:
```
# These functions are part of the python-colorama module
# They have been adjusted slightly for LinkChecker
```

Oh that is just lovely.  :(

Comment (anarcat) [reply linkcheck/colorama.py:30]: ugh another ticket?

Comment (cjmayo) [reply linkcheck/colorama.py:30]: > ugh another ticket?

It is for Windows only:

ansicolor.py:62
```py
if os.name == 'nt':
    from . import colorama
```

**Pull request merged**